petcare-enterprise-apps/docs/PETDESK-B2C-LOCAL.md

```diff
@@ -0,0 +1,179 @@
+# Run PetDesk B2C Application locally
+
+## Prerequisites:
+1. Install Ballerina 2201.5.0 https://dist.ballerina.io/downloads/2201.5.0/ballerina-2201.5.0-swan-lake-macos-arm-x64.pkg
+2. Install Node 16 LTS (Tested in v16.13.0).
+3. Configure choreo webhook for [Asgardeo user registration event](https://wso2.com/asgardeo/docs/guides/asgardeo-events/#implement-business-use-cases-for-asgardeo-events). Use [asgardeo_registration_webhook](/petcare-enterprise-apps/b2c/webhooks/asgardeo_registration_webhook/).
+
+When deploying the webhook through choreo, provide the salesforce related configuration by getting them using this [guide](#setup-salesforce-account-guide).
+
+
+## Create an Application in WSO2 Identity Server
```

Comment:
Let's say asgardeo as tutorial is formed around asgardeo here

Reply:
Added with 060ec807c496a52bde3a462dea3f0ec7c480360f